feature-configs/demo/dpdk/is_ready.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+oc -n dpdk-testing get build -l app=s2i-dpdk
+
+oc -n dpdk-testing wait build --all --for condition=Complete --timeout 1s
+
+oc get dc s2i-dpdk-app
```

Comment:
Why we need this?

Reply:
To validate there is a deploymentConfig (if there is not it will return an exit code -1)